src/Files.App/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -287,47 +287,59 @@ public ColumnsViewModel ColumnsViewModel
 		private static LayoutPreferences GetLayoutPreferencesForPath(string folderPath)
 		{
 			IUserSettingsService userSettingsService = Ioc.Default.GetRequiredService<IUserSettingsService>();
-			if (userSettingsService.FoldersSettingsService.EnableOverridingFolderPreferences)
+
+			var layout = LayoutPreferences.DefaultLayoutPreferences;
+
+			if (userSettingsService.FoldersSettingsService.EnableOverridingFolderPreferences || userSettingsService.LayoutSettingsService.EnableOverridingSortingPreferences)
```

Comment:
You're using these two variables two times. One here and one in l.302 and l.323.
You'd be better off saving it inside variables to not access them two times.